gpkit/constraints/tight.py

```diff
@@ -35,7 +35,7 @@ def process_result(self, result):
                         leftsubbed = constraint.left.sub(variables).value
                         rightsubbed = constraint.right.sub(variables).value
             if rel_diff >= self.reltol:
-                msg = ("Constraint [%.30s...] is not tight because "
+                msg = ("Constraint [%.200s...] is not tight because "
                        "the left hand side evaluated to %s but "
```

Comment:
Hmmk! For long string / variable names, might it be better to print a certain number of variables instead? Or we could print 100 characters on the left, the operator, then 100 characters on the right?